include/wx/qt/window.h

```diff
@@ -221,6 +221,7 @@ class WXDLLIMPEXP_CORE wxWindowQt : public wxWindowBase
     // itself.
     virtual QWidget* QtGetParentWidget() const { return GetHandle(); }
 
+    virtual bool EnableTouchEvents(int eventsMask);
```

Comment:
I think this will trigger clang warnings about inconsistent use of `override`. wxQt should be updated to use it everywhere where it makes sense first, before starting to use it anywhere...

Reply:
There are already lots of `wxOVERRIDE` in this header, why would this one be different?

Reply:
Oops, yes, you're totally right, sorry. I've only looked at `QtGetParentWidget()` above and recalled my previous problems with adding `wxOVERRIDE` to the (other) wxQt headers which resulted in this problem without realizing that this function wasn't actually overriding anything but was a new virtual in this class.

Sorry again, please disregard the comment above (not deleting it just to avoid confusion).

Reply:
Thanks, I will come back to this soon.  We are fixing the hard coded bits first.